odxtools/compumethods/texttablecompumethod.py

```diff
@@ -39,8 +39,10 @@ def __is_internal_in_scale(self, internal_value, scale: CompuScale):
         if scale.lower_limit is not None and not scale.lower_limit.complies_to_lower(
                 internal_value):
             return False
-        if scale.upper_limit is not None and not scale.upper_limit.complies_to_upper(
-                internal_value):
+        # If no UPPER-LIMIT is defined
+        # the COMPU-SCALE will be applied only for the value defined in LOWER-LIMIT
+        upper_limit = scale.upper_limit or scale.lower_limit
```

Comment:
what happens if `upper_limit` is `0`?
```suggestion
        upper_limit = scale.upper_limit
        if upper_limit is None:
            upper_limit = scale.lower_limit
```

Reply:
scale.lower_limit have type Limit, not int, so even if `scale.lower_limit.value = 0`, `scale.lower_limit` will still be truthy